components/elective/EnrollmentPrintoutHeader.tsx

```diff
@@ -37,7 +38,13 @@ const EnrollmentPrintoutHeader: StylableFC<{
       </div>
 
       {/* Class number and Advisors */}
-      <div className="flex flex-row pl-16">
+      <div className="flex flex-row">
+        <span
+          className={cn(`-mt-2 ml-4 mr-14 inline-block self-end
```

Comment:
The use of cn to keep lines under 80 characters makes it good for cleanliness and readable code!